source/assets/js/components/fef-a11y.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Some browsers do not fully support in-page links. While they may visually shift focus to the location
+ * of the target or named anchor for the “skip” link, they do not actually set keyboard focus to this location.
+ *
+ * https://www.bignerdranch.com/blog/web-accessibility-skip-navigation-links/
+ */
+
+
+export function init() {
+    // flag link to inform about manipulation of the link
+    $('.js-skiplink').attr('data-skiplink', 'true');
+
+    $('.js-skiplink').click(function() {
```

Comment:
mötzle uf hohem niveau:
* `$('.js-skiplink')` zwischenspeichern als variable
* `.on('click', () => ...)` statt `.click(...)` verwenden

Reply:
Skiplink ist jetzt zwischengespeichert.

Ist auf on('click' geändert, jedoch ohne Arrow-Function, da ich innerhalb der Funktion das this benötige.